config/routes.rb

```diff
@@ -8,4 +8,7 @@
   resources :likes
   resources :photos
 
+
```

Comment:
Make sure you get rid of the duplicate `devise_for :users`, it is in this file twice!